dash_bio/utils/pdbParser.py

```diff
@@ -1,86 +1,94 @@
-import sys
-import numpy as np
 import parmed as pmd
 import re
 import json
 import os
 from shutil import copy2
 
-def createData(pdbPath):
-# def createData(pdbPath):
-    '''
-    Function to parse the protein data bank (PDB) file to generate the input modelData
 
-    @param pdbPath
+def create_data(pdb_path):
+    """
+    Function to parse the protein data bank (PDB) file to generate
+    the input modelData

```

Comment:
```suggestion
    input modelData
```
or, alternatively, "the modelData input" (even though it's not clear to me that it would be clear to all--what that input is)